src/Pangraph/Internal/Graph.hs

```diff
@@ -0,0 +1,206 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Pangraph.Internal.Graph (
+-- Abstract Types
+Pangraph,
+MalformedEdge,
+Edge,
+Vertex,
+Attribute, -- A type alias for (Key, Value)
+Key,
+Value,
+VertexID,
+EdgeID,
+
+-- Constructors
+makePangraph,
+makeEdge,
+makeVertex,
+
+-- Pangraph Getters
+edges,
+vertices,
+mapEdges,
```

Comment:
Let's drop this for now. Remember, we go for the simplest API that works. No need for `mapEdges` or `mapVertices`.